### ISSUE TITLE
Virtual Cockpit is reset

### ISSUE THREAD

**Issue by etrondriver** [electrichasgoneaudi/etron-issues#8]
## Description

[Virtual Cockpit](https://electrichasgoneaudi.net/models/e-tron/technology/uiandoperations/virtualcockpit/)  in [Audi e-tron](https://electrichasgoneaudi.net/models/e-tron/) has 2 or 3 modes depending on if the car is equipped With [Virtual Cockpit plus](https://electrichasgoneaudi.net/models/e-tron/technology/uiandoperations/virtualcockpit/#virtual-cockpit-plus) or not.

The modes are
- Standard
- e-tron
- sport

Owners report that Virtual Cockpit is reset to the standard mode. 

## From Manual
![image](https://user-images.githubusercontent.com/59776765/72211061-d5988b80-34c4-11ea-87c0-b3ceb23ced9d.png)

## How to fix
There is no know fix for this

## Online incident reports
[Audi e-tron Norge September 9. 2019](https://www.facebook.com/groups/361111001113524/permalink/494080567816566/)
[Audi e-tron Norge 2020](https://www.facebook.com/groups/361111001113524/permalink/591504491407506/)
[Audi etron Enthusiasts September 24. 2021 (with information about possible SW fix)](https://www.facebook.com/groups/audietronenthusiasts/permalink/2988774444703689/)

**Comment by pultzner** [comment]
Yes, this happens to me also without known reason

**Comment by deltamik** [comment]
VC reset #1

**Comment by markusv** [comment]
Yes this happened to me as well. Møller arrogantly suggested I had to use the wrong key, since this is stored in the user profile. Anyhow this stopped happening by itself. I have no idea why

**Comment by pultzner** [comment]
I only use digital key and not physical key and it still happens

**Comment by markusv** [comment]
I am 100% sure that I dit not use the wrong key. The seats position is also stored with the key. And I can't even get in the car with the seats in my wifes position. So that is just Møller always blaming the user. they told me they would check with the factory. But I never heard back from them. Since this stopped happening I have not followed them up on this issue.

**Comment by RemiProvost** [comment]
I also happends for me now and then. Very frustrating, especially because I can’t change back to my preferences until the car car has been fully turned off and “restarted”.
It simple just doesn’t respond to input changes while in this bad behavior mood 😅

I always use my own key, my wife has the other.

**Comment by wataabirs** [comment]
I had this problem also, but I found a possible workaround: after adjusting the VC to the e-tron mode I have saved my seat position with the seat memory function and now the VC stays e-tron mode (for a couple of weeks already). Before this it reverted to the Standard mode every couple of hours/days.
